test/system_test.py

```diff
@@ -0,0 +1,135 @@
+
+    @pytest.mark.usefixtures("_extract_radiology_reports")
+    def test_ftps_parquet_upload(self):
+        """The copied parquet files"""
+        time.sleep(2)
+        assert TestFtpsUpload.expected_public_parquet_dir.exists()
+        assert (
+            TestFtpsUpload.expected_public_parquet_dir
+            / "omop"
+            / "public"
+            / "PROCEDURE_OCCURRENCE.parquet"
+        ).exists()
+        assert (
+            TestFtpsUpload.expected_public_parquet_dir / "radiology" / "radiology.parquet"
+        ).exists()
+
+    @pytest.mark.usefixtures("_extract_radiology_reports")
+    def test_ftps_dicom_upload(self):
+        """Test whether DICOM images have been uploaded"""
+        SECONDS_WAIT = 5
+
+        zip_files = []
+        for seconds in range(0, 121, SECONDS_WAIT):
+            zip_files = list(TestFtpsUpload.expected_output_dir.glob("*.zip"))
+            logging.info("Waited for %s seconds. glob_list: %s", seconds, zip_files)
+            if len(zip_files) == 2:
+                break
+            sleep(SECONDS_WAIT)
+
+        # We expect 2 DICOM image studies to be uploaded
+        assert len(zip_files) == 2
+
+
+def _wait_for_stable_orthanc_anon(seconds_max, seconds_interval) -> None:
+    """
+    Query the orthanc-anon REST API to check that the correct number of instances
+    have been received.
+    If they haven't within the time limit, raise a TimeoutError
+    """
+    for seconds in range(0, seconds_max, seconds_interval):
+        instances_cmd = shlex.split(
+            "docker exec system-test-orthanc-anon-1 "
+            'curl -u "orthanc_anon_username:orthanc_anon_password" '
+            "http://orthanc-anon:8042/instances"
+        )
+        instances_output = subprocess.run(instances_cmd, capture_output=True, check=True, text=True)  # noqa: S603
+        instances = json.loads(instances_output.stdout)
+        logging.info("Waited for %s seconds, orthanc-anon instances: %s", seconds, instances)
+        if len(instances) == 2:
+            return  # success
+        sleep(seconds_interval)
+    raise TimeoutError
+
+
+@pytest.mark.usefixtures("_setup_pixl_cli")
+def test_ehr_anon_entries():
+    """
+    ./scripts/check_entry_in_orthanc_anon_for_2_min.py
+    ./scripts/check_entry_in_pixl_anon.sh
+    """
+    # Because we have to wait for a stable study, poll for 2 minutes
+    _wait_for_stable_orthanc_anon(121, 5)
+
+    # This was converted from old shell script - might be better to check the data itself though?
+    sql_command = "select * from emap_data.ehr_anon"
+    cp = run_subprocess(
+        [
+            "docker",
+            "exec",
+            "system-test-postgres-1",
+            "/bin/bash",
+            "-c",
+            f'PGPASSWORD=pixl_db_password psql -U pixl_db_username -d pixl -c "{sql_command}"',
+        ],
+        Path.cwd(),
+    )
+    assert cp.stdout.decode().find("(2 rows)") != -1
+
+
+def test_max_storage_in_orthanc_raw():
```

Comment:
Imagine you're aware that this needs to be done, but just in case. I'm not super sure how useful these tests are, being able to create MRIs is useful so if its a pain with the order then don't spend long on it and drop them instead

Reply:
Sorry I'm a bit lost, how is this connected to MRIs?

Reply:
The tests create MRIs and send them into orthanc raw